panimg/models.py

```diff
@@ -156,6 +156,8 @@ class PanImg:
     study_description: str = ""
     series_description: str = ""
     segments: Optional[FrozenSet[int]] = None
+    min_voxel_value: Optional[int] = None
+    max_voxel_value: Optional[int] = None
```

Comment:
These two will need to be stored on the GC side. Is this necessary? I thought that we only needed to set `segments` for TIFF files?

Reply:
It was decided in the pitch to use the min/max value and a check to see if it's contiguous to validate segmentation compatibility. In order to do that we need to set these properties on the image model.

Reply:
Yes but then we get duplicate information in the database, which could go out of sync.

Reply:
But in panimg `segments` is always empty for tiff images. Or which duplicate information do you mean?

Reply:
Yes, so if the tiff file contains integer data set the segments based on the min voxel value, max voxel value and the possible values in between, keep it `None` otherwise.

Reply:
These two lines can be removed now, right?